docs/faq.md

```diff
@@ -95,3 +95,38 @@ in real time.
 
 Conversely, if the kubelet is still in a retry cycle trying to launch a Pod with a `SharedConfigMap` or `SharedSecret` reference, if now resolved permission issues were the only thing preventing
 a mount, the mount should then succeed.  Of course, as kubelet retry vs. controller re-list is the polling mechanism, and it is more frequent, the change in results would be more immediate in this case.
+
+# Other Secret Providers/Operators
+
+## Can't I do the same thing with the External Secrets Operator?
+
+For Secrets, yes. The [External Secrets Operator](https://external-secrets.io) can
+share Secrets through its [Kubernetes provider](https://external-secrets.io/latest/provider/kubernetes/).
+However, there are several technical differences between this and External Secrets
+operator - see the "Feature Comparison" section below.
+
+## Can this be accomplished with the Secret Store CSI Driver?
+
+Not at present. The [Secret Store CSI Driver](https://secrets-store-csi-driver.sigs.k8s.io)
+also allows secrets to be synced and mounted with `CSI` volumes. A separate [provider](https://secrets-store-csi-driver.sigs.k8s.io/providers)
+is responsible for fetching and syncing the referenced data. As of this writing, no
+provider syncs data that explicitly exists within a Kubernetes cluster.
+
+Early in this project's history, we considered implementing a provider for the
+Secret Store CSI Driver. This was declined in favor the current RBAC model and
+implementing the CSI interfaces required for `Ephemeral` volume lifecycle mode.
```

Comment:
We often talked about the Secret Store and Shared resources being complementary:

- use Secret Source to bring in the credentials from a source outside the Kubernetes cluster, but into one namespace

-   use Shared Secrets to allow other users to leverage the credentials brought in by Secret Store, without having to copy the secret, or define a Secret Store in all the namespaces that need those credentials.

IMO this complimentary relationship exists regardless of whether we ever implemented a provider for Secret Store.

Reply:
Good points - I tried to address as follows:

- Adding an intro paragraph in the section for the related Secret Store and External Secrets Operator projects
- Adding third FAQ - "Can I deploy this alongside the Secret Store CSI Driver?"